### PR TITLE
Added more commands into game

### PR DESCRIPTION
**Pull request by mmm1245**

### CODE REVIEW

File: core/src/mindustry/core/NetServer.java [hunk 1]
```diff
@@ -300,6 +303,295 @@ private void registerCommands(){
...
+                player.sendMessage("[scarlet]You can't specify arguments.");
+                return;
+            }
+
+            player.sendMessage("[orange]Items:");
+
+            for(int i = 0;i < 10000;i++){
+                Item it = content.item(i);
+                if(it == null)
+                    break;
```
Comment (Quezler) [inline core/src/mindustry/core/NetServer.java]: use `{ }` or put them on the same line

File: core/src/mindustry/core/NetServer.java [hunk 2]
```diff
@@ -300,6 +303,295 @@ private void registerCommands(){
...
+
+                player.sendMessage("[white]" + i + ":" + it.localizedName);
+            }
+            
+        });
+
+        clientCommands.<Player>register("statuslist", "", "Gives you list of all statuses.", (args, player) -> {
+            
+
```
Comment (Quezler) [inline core/src/mindustry/core/NetServer.java]: double newline

File: core/src/mindustry/core/NetServer.java [hunk 3]
```diff
@@ -300,6 +303,295 @@ private void registerCommands(){
...
+            player.sendMessage("[white]boss");
+            player.sendMessage("[white]burning");
+            player.sendMessage("[white]corroded");
+            player.sendMessage("[white]freezing");
+            player.sendMessage("[white]melting");
+            player.sendMessage("[white]overdrive");
+            player.sendMessage("[white]shielded");
+            player.sendMessage("[white]shocked");
+            player.sendMessage("[white]tarred");
+            player.sendMessage("[white]wet");
```
Comment (Quezler) [inline core/src/mindustry/core/NetServer.java]: you should probably loop through them from the code, and not hardcode them

File: core/src/mindustry/core/NetServer.java [hunk 4]
```diff
@@ -300,6 +303,295 @@ private void registerCommands(){
...
+                player.sendMessage("[scarlet]You must specify 2 arguments or use list.");
+                return;
+            }
+
+            if(!(Strings.canParseInt(args[1]))){
+                player.sendMessage("[scarlet]Duration arguments is not number.");
+                return;
+            }
+
+
```
Comment (Quezler) [inline core/src/mindustry/core/NetServer.java]: newlines

File: core/src/mindustry/core/NetServer.java [hunk 5]
```diff
@@ -300,6 +303,295 @@ private void registerCommands(){
...
+            } else if(args[0].equals("shocked")){
+                player.applyEffect(StatusEffects.shocked, Integer.parseInt(args[1]));
+            } else if(args[0].equals("tarred")){
+                player.applyEffect(StatusEffects.tarred, Integer.parseInt(args[1]));
+            } else if(args[0].equals("wet")){
+                player.applyEffect(StatusEffects.wet, Integer.parseInt(args[1]));
+            } else {
+                player.sendMessage("[scarlet]Effect was not found. Type [orange]applystatusme list[white] to list all effects");
+            }
+
```
Comment (Quezler) [inline core/src/mindustry/core/NetServer.java]: a switch and a variable would do good here

File: core/src/mindustry/core/NetServer.java [hunk 6]
```diff
@@ -300,6 +303,295 @@ private void registerCommands(){
...
+            return;
+            
+            
+        });
+
+        clientCommands.<Player>register("cheats", "", "Prints whenever you are cheating.", (args, player) -> {
```
Comment (Quezler) [inline core/src/mindustry/core/NetServer.java]: empty arguments

File: core/src/mindustry/core/NetServer.java [hunk 7]
```diff
@@ -300,6 +303,295 @@ private void registerCommands(){
...
+            if(Vars.state.rules.cheating){
+                player.sendMessage("Cheats are turned [green]on.");
+            } else {
+                player.sendMessage("Cheats are turned [scarlet]off.");
+            }
+        });
+
+        clientCommands.<Player>register("cheatson", "", "Turns on cheating mode(admin-only). [yellow]WARNING: [orange]cheatson [yellow] cannot be reverted.", (args, player) -> {
```
Comment (Quezler) [inline core/src/mindustry/core/NetServer.java]: empty arguments

File: core/src/mindustry/game/Rules.java [hunk 8]
```diff
@@ -14,6 +14,8 @@
  */
 @Serialize
 public class Rules{
+    /** Whether allow cheating */
```
Comment (Quezler) [inline core/src/mindustry/game/Rules.java]:
```suggestion
    /** Whether to allow cheating */
```

**Comment by Anuken** [comment]
I don't think cheat commands should come with the game. These things can already be done through the server console with `js` if needed.

**Pull request closed without merging**